samza-sql/src/main/java/org/apache/samza/sql/avro/AvroRelConverter.java

```diff
@@ -99,17 +99,6 @@ public SamzaSqlRelMessage convertToRelMessage(KV<Object, Object> samzaMessage) {
         new SamzaSqlRelMsgMetadata(0L, 0L));
   }
 
-  /**
-   * Create a SamzaSqlRelMessage for the specified key and Avro record using the schema from the Avro record.
-   *
-   */
-  public static SamzaSqlRelMessage convertToRelMessage(Object key, IndexedRecord record, Schema schema) {
```

Comment:
although convertToRelMessage is used in samza repo directly, it is invoked by others, e.g. BrooklinRelConverter and other xxxRelconverter.